src/pages/HomePage/ContactSection.jsx

```diff
@@ -0,0 +1,148 @@
+import React, { useEffect, useState } from "react";
+import {
+  Typography,
+  Box,
+  Link,
+  IconButton,
+  Divider,
+  Container,
+  Grid,
+  useTheme,
+  Stack,
+} from "@mui/material";
+import EmailIcon from "@mui/icons-material/Email";
+import { FaInstagram } from "react-icons/fa";
+
+const ContactSection = () => {
+  const theme = useTheme();
+  const SPACING = 2;
```

Comment:
This is not needed - we shouldnt hard code theme varibales like this. you can put it in the component.

Reply:
removed